docs_src/async_sql_databases/sql_app/tests/test_sql_app.py

```diff
@@ -4,8 +4,9 @@
 from ..database import Base
 from ..main import app, get_db
 
-SQLALCHEMY_DATABASE_URL = "sqlite:///./test.db"
+SQLALCHEMY_DATABASE_URL = "sqlite+aiosqlite:///./test.db"
 
+# connect_args is ONLY for sqlite, remove if using PostgreSQL or other.
```

Comment:
Maybe following the analogous: `docs_src/sql_databases/sql_app/database.py`.

I'm too tired, not going to give you the link, sorry. :kissing:

Reply:
Link: https://github.com/tiangolo/fastapi/blob/master/docs_src/sql_databases/sql_app/database.py

This PR is analogous of that, but seeing how a few people here got confused about it (https://github.com/tiangolo/fastapi/pull/2331#issuecomment-801461215) felt like it was a worth while comment? Not that strong on it so can remove.